ext/debian/puppet.conf

```diff
@@ -0,0 +1 @@
+../../conf/puppet.conf
```

Comment:
Will someone followup with the changes made to hiera that also introduced symlinks? @MikaelSmith pointed them out.